openedx/edx-ora2
# fix: submission too long has the wrong message

**TL;DR -** [ Fix the error message for when an ORA submission is too long ]

JIRA: [AU-559](https://openedx.atlassian.net/browse/AU-559)

**What changed?**

- Change the error message when an ORA is too long to be about that instead of multiple submissions
- Add a note about autocorrect

**Developer Checklist**


**Testing Instructions**

- In an ORA, submit something super long. Like ... the entire text of moby dick https://www.gutenberg.org/files/2701/2701-h/2701-h.htm and see the new error message

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
